model_analyzer/config/input/config_defaults.py

```diff
@@ -41,8 +41,6 @@
 DEFAULT_RUN_CONFIG_MAX_CONCURRENCY = 1024
 DEFAULT_RUN_CONFIG_MAX_INSTANCE_COUNT = 5
 DEFAULT_RUN_CONFIG_SEARCH_DISABLE = False
-DEFAULT_RUN_CONFIG_MAX_PREFERRED_BATCH_SIZE = 16
```

Comment:
Why is this showing up? Are we not properly rebased?

Reply:
Yes. I've reset, squashed and rebased properly.